src/components/views/elements/SearchWarning.tsx

```diff
@@ -71,7 +71,7 @@ export default function SearchWarning({ isRoomEncrypted, kind }: IProps): JSX.El
 
     let text = null;
     let logo = null;
-    if (desktopBuilds.get("available")) {
+    if (desktopBuilds?.get("available")) {
```

Comment:
I don't think this is necessary -- SdkConfig.getObject already falls back to the default desktopBuilds config if none is provided, so it can never be null. If anything, we should fix the return types of SdkConfig.get and getObject to account for the fact that they will never return null if the default config has that key.

Reply:
`    const val = SdkConfig.get(key, altCaseName);
        if (val !== null && val !== undefined) {
            return new SnakedObject(val);
        }
`
return types of SdkConfig.get and SdkConfig.getObject are already handled for undefined and null.  Can you please tell what exactly needs to be fixed then.

Reply:
Slightly difficult to say, you pull request provides very little context as to what you're trying to achieve and why you're doing it

Reply:
https://github.com/vector-im/element-web/issues/23842
I forgot to attach this issue in my PR. My PR resolves this issue.